pkg/controller/job/job_controller.go

```diff
@@ -718,16 +718,12 @@ func (jm *Controller) syncJob(ctx context.Context, key string) (forget bool, rEr
 		metrics.JobSyncNum.WithLabelValues(completionMode, result, action).Inc()
 	}()
 
-	var expectedRmFinalizers sets.String
-	var uncounted *uncountedTerminatedPods
-	if hasJobTrackingAnnotation(&job) {
-		klog.V(4).InfoS("Tracking uncounted Pods with pod finalizers", "job", klog.KObj(&job))
-		if job.Status.UncountedTerminatedPods == nil {
-			job.Status.UncountedTerminatedPods = &batch.UncountedTerminatedPods{}
-		}
-		uncounted = newUncountedTerminatedPods(*job.Status.UncountedTerminatedPods)
-		expectedRmFinalizers = jm.finalizerExpectations.getExpectedUIDs(key)
+	klog.V(4).InfoS("Tracking uncounted Pods with pod finalizers", "job", klog.KObj(&job))
```

Comment:
With only supporting uncounted pods with finalizers, should we just remove this log?

Reply:
sure thing :)